data/event-schemas/schema/m.room.topic.yaml

```diff
@@ -1,7 +1,14 @@
 ---
 allOf:
   - $ref: core-event-schema/state_event.yaml
-description: 'A topic is a short message detailing what is currently being discussed in the room.  It can also be used as a way to display extra information about the room, which may not be suitable for the room name. The room topic can also be set when creating a room using `/createRoom` with the `topic` key.'
+description: |- 
+    A topic is a short message detailing what is currently being discussed in the room.
+    It can also be used as a way to display extra information about the room, which may not
+    be suitable for the room name.
+    The room topic can also be set when creating a room using `/createRoom` with the `topic` key.'
+    
+    If a room has an `m.room.topic` event with an absent, null, or empty `topic`
+    field, it should be treated the same as a room with no `m.room.topic` event.
```

Comment:
While it's a great clarification in itself, it would be even greater to further clarify what happens when an event with an absent/null/empty `topic` comes upon a previous event with non-falsey `topic`. I guess the room topic should be reset to the as-if-no-topic state?

Reply:
Hm, I suppose I thought that was the implication as with the `m.room.name` description => clients should use the latest state to render the room, and this description tells clients to assume the topic isn't set.

I'm struggling to find words that doesn't sound a bit repetitive?

Reply:
better?